foodhub_custom/views/templates.xml

```diff
@@ -0,0 +1,7 @@
+<odoo>
+    <template id="hide_product_terms" inherit_id="website_sale.product" name="Hide Terms and Conditions">
+        <xpath expr="//div[@id='product_details']/p[@class='text-muted']" position="attributes">
+            <attribute name="class">hidden</attribute>
```

Comment:
Dans odoo, on utilise plutôt l'attribut `invisible`, par exemple:
```
                <field name="report_id" position="attributes">
                    <attribute name="invisible">1</attribute>
                </field>
```

Reply:
You cannot do that in qweb, unfortunatly. :/ Only xpath. :)

Reply:
Ha mais attends, je parlais pas du Xpath, je parlais de l'attribut utilisé! Genre ceci:

```xml
<xpath expr="//div[@id='product_details']/p[@class='text-muted']" 
       position="attributes">
    <attribute name="invisible">1</attribute>
</xpath>
```

Reply:
(oups)